paper/paper.md

```diff
@@ -0,0 +1,78 @@
+
+The library can also be used to evaluate statistics from a sample. We can estimate the parameters of a probabilistic model including extreme value models (e.g. Generalized Extreme Value distribution and Generalized Pareto distribution) and validate them using fitting tests (e.g. Kolmogorov-Smirnov, Anderson-Darling, etc.) or using graphical tests (e.g. QQ-plot, Kendall plot). Non parametric models can be used based on a sample including kernel density estimation or Bernstein copula. For a given model, variables can be prioritized using sensitivity analysis methods (e.g. SRC, Sobol' and HSIC indices). A function can also be approximated by a meta-model, including linear model, polynomial chaos expansion, Gaussian process regression. A unified interface of optimization algorithms (NLOpt, Ipopt, PAGMO, Bonmin, Dlib, Ceres, Cminpack, Cobyla, etc.) is available and used in particular in the context of parametric model calibration with Bayesian or non-Bayesian methods. Since many probabilistic methods require the evaluation of an integral, numerical integration methods are available by sampling or quadrature (if necessary, adaptive). The provided quadrature methods are the multivariate Gaussian quadrature (e.g. Gauss Legendre quadrature), Gauss-Kronrod, Smolyak quadrature and other algorithms including algorithms from Cuba's library.  The library provides full or compressed linear algebra capabilities (in particular for covariance matrices). Numerous reliability algorithms are implemented for probability calculation and quantification of rare events (e.g. FORM-SORM, subset sampling). Other reliability algorithms such as Non parametric Adaptive Importance Sampling (NAIS) and Cross-Entropy are available.
+
+## Usages
+The OpenTURNS library is used for research and engineering studies. OpenTURNS is used for higher education at universities (DTU Denmark) and engineering schools (ENSTA, ENPC, Centrale Nantes, Sigma Clermont, Sup Galilée).
+
+The library is presented in several publications, e.g. `@Baudin:2016`. For example, calibration features of the library are introduced in `@Baudin:2021`. External modules such as the benchmarking module `otbenchmark` is introduced in `@Fekhari:2021`. Numerous application articles are based on OpenTURNS (e.g. `@Dutfoy:2023` on seismic risk assessment using the Gutenberg-Richter law).
+
+The Persalys graphical user interface provides uncertainty quantification features using OpenTURNS (see www.persalys.fr) . Other Persalys-derived graphical tools are also based on OpenTURNS (see `@Bryla:2022`). 
+
+A User's day meeting has been organized each year since 2008. The goal of this meeting is to present the use of OpenTURNS in engineering or research uncertainty quantification studies.
```

Comment:
it could be interesting to say that we also present to the community the novelty in terms of developments since the previous User's day edition.
for instance :
"of this meeting is" to introduce to the community the new developments from the previous releases and to present

Reply:
I added a new sentence: "It's also an opportunity to present the new features of each version of the library, as well as the roadmap.". So this is fixed.